### ISSUE TITLE
Rayvista crashes R session

### ISSUE THREAD

**Issue by magnusnil** [h-a-graham/rayvista#40]
When I run the first example from https://github.com/h-a-graham/rayvista, the R session crashes. Tried with R 4.3.0 and RStudio on Windows 10.

**Comment by h-a-graham** [comment]
Thanks for reporting - I might need a bit more info... 

Do the examples on https://github.com/tylermorganwall/rayshader work for you? 

What is your `sessionInfo()`?

**Comment by magnusnil** [comment]
Yes, I tried the first example at https://github.com/tylermorganwall/rayshader and it works fine.

sessionInfo():

R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=Swedish_Sweden.utf8  LC_CTYPE=Swedish_Sweden.utf8    LC_MONETARY=Swedish_Sweden.utf8 LC_NUMERIC=C                    LC_TIME=Swedish_Sweden.utf8    

time zone: Europe/Stockholm
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.3.0  tools_4.3.0     rstudioapi_0.14
